packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -79,6 +82,7 @@ const LinkCore = React.forwardRef(function LinkCore(
             size="small"
             style={[styles.endIcon, styles.centered]}
             testId="external-icon"
+            aria-label={`(${externalIconAriaLabel})`}
```

Comment:
I think the parentheses will need to be included when strings get translated since other languages might handle it differently! So consumers would need to pass in a translated string for `(opens in a new tab)` rather than `opens in a new tab`